release notes/v0.39.0.md

```diff
@@ -0,0 +1,32 @@
+- [#2509](https://github.com/grafana/k6/pull/2509): Add `clear()` and `delete()` methods to the [`CookieJar` object](https://k6.io/docs/javascript-api/k6-http/cookiejar/) from the [`k6/http`](https://k6.io/docs/javascript-api/k6-http/) module. Thanks, @Maksimall89!
+- [#2282](https://github.com/grafana/k6/pull/2282): The iterations/s precision in the UI progress bars was increased. Thanks, @m3hm3t!
+- [#2568](https://github.com/grafana/k6/pull/2568): Added more descriptive error messages when there were problems with parsing a config file.
+
+## Bug fixes
+
+- [#2523](https://github.com/grafana/k6/pull/2523): Fixed a gRPC marshaling error when `any.proto` was used for a type. Thanks, @Flowersea!
+- [#2534](https://github.com/grafana/k6/pull/2534): Fixed the return type of [`Selection.map()`](https://k6.io/docs/javascript-api/k6-html/selection/selection-map) from the [`k6/html` module](https://k6.io/docs/javascript-api/k6-html/) to the correct object types instead of a forced array of strings.
+- [#2502](https://github.com/grafana/k6/pull/2502): k6 will now wait for scenario executors to fully finish before updating their final progress in the UI, preventing misleading red crosses ([#2500](https://github.com/grafana/k6/issues/2500)).
+- [#2524](https://github.com/grafana/k6/pull/2524): Fixed a bug where GoError string contained missing URL values ([#2537](https://github.com/grafana/k6/issues/2537)).
+- [#2530](https://github.com/grafana/k6/pull/2530): Fixed a wrong error message on remote resolution.
+- [#2542](https://github.com/grafana/k6/pull/2542): Fixed a bug where `Rate` metric and sub-metric values were shown as `NaN` in the end-of-test summary if there were no measured values for them during the test run.
+- [#2558](https://github.com/grafana/k6/pull/2558): Fixed a panic when trying to set the value of a [`vu.tags` element from `k6/execution`](https://k6.io/docs/javascript-api/k6-execution/#vu) to `null` or `undefined`.
+- [#2567](https://github.com/grafana/k6/pull/2567): Fixed a panic when trying to access some [`k6/execution`](https://k6.io/docs/javascript-api/k6-execution/) properties outside of a VU context, e.g. trying to access `execution.scenario` in `setup()`.
+
+## Maintenance and internal improvements
+
+- [#2550](https://github.com/grafana/k6/pull/2550): Updated the used Go version to 1.18.
+- [#2524](https://github.com/grafana/k6/pull/2524), [#2551](https://github.com/grafana/k6/pull/2551), [#2552](https://github.com/grafana/k6/pull/2552), [#2553](https://github.com/grafana/k6/pull/2553), [#2554](https://github.com/grafana/k6/pull/2554), [#2555](https://github.com/grafana/k6/pull/2555): Updates of various Go dependencies in k6
```

Comment:
```suggestion
- [#2524](https://github.com/grafana/k6/pull/2524), [#2551](https://github.com/grafana/k6/pull/2551), [#2552](https://github.com/grafana/k6/pull/2552), [#2553](https://github.com/grafana/k6/pull/2553), [#2554](https://github.com/grafana/k6/pull/2554), [#2555](https://github.com/grafana/k6/pull/2555): Updated various Go dependencies in k6
```

Reply:
Resolved by the latest [force push](https://github.com/grafana/k6/compare/9d682d82437985265e135cd6a24080be0b007dbd..6f763113a21481f8b3d72ac5496366cd5ffbd982)